packages/lib/src/vis/heatmap/utils.ts

```diff
@@ -40,25 +25,14 @@ export const GRADIENT_RANGE = range(
   GRADIENT_PRECISION
 );
 
-export const TEXTURE_TYPE_BY_DTYPE: Record<
-  DataType<TextureTypedArray>,
-  TextureDataType
-> = {
-  int8: ByteType,
-  int16: ShortType,
-  int32: IntType,
-  uint8: UnsignedByteType,
-  uint16: UnsignedShortType,
-  uint32: UnsignedIntType,
-  float32: FloatType,
+const SCALE_FNS: Record<ScaleType, (val: number) => number> = {
+  [ScaleType.Linear]: (val) => val,
+  [ScaleType.Log]: Math.log10,
+  [ScaleType.SymLog]: (val) => Math.sign(val) * Math.log10(1 + Math.abs(val)),
+  [ScaleType.Sqrt]: Math.sqrt,
+  [ScaleType.Gamma]: (val) => val,
```

Comment:
This is where the scale functions end up. `SCALE_FNS` is then used in `scaleDomain()` below.